### PR TITLE
add working group of sequence

### PR DESCRIPTION
**Pull request by AilinKid**
Add working group of Sequence.

### CODE REVIEW

File: working-groups/wg-tidb-sequence.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# TiDB Sequence Working Group
+
+## Goals
+
+Support Sequence in TiDB
+
+* Many customers could probably make do with using Auto_Increment as unique id generator,
```
Comment (zz-jason) [inline working-groups/wg-tidb-sequence.md]:
```suggestion
* Many customers could probably make do with using `AUTO_INCREMENT` as unique id generator,
```

File: working-groups/wg-tidb-sequence.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* Meeting Notes: [Meeting Notes and Agenda](https://docs.google.com/document/d/1h09ST495ah9d0KJIWHn4w_YZVJlxPvoa4-5FVOF0sac/edit?usp=sharing)
+
+## Mentors
+
+* [winkyao](http://github.com/winkyao), [PingCAP](https://github.com/pingcap)
+
+## Organizers
+
+* [ailinkid](https://github.com/AilinKid), [PingCAP](https://github.com/pingcap)
+* [andrewdi](https://github.com/AndrewDi), [Personal](pandipd@outlook.com)
```
Comment (zz-jason) [inline working-groups/wg-tidb-sequence.md]:
```suggestion
* [andrewdi](https://github.com/AndrewDi), [Personal](mailto:pandipd@outlook.com)
```

File: working-groups/wg-tidb-sequence.md (hunk 1 above)
Comment (zz-jason) [inline working-groups/wg-tidb-sequence.md]: @dcalvin Would you like to help us polish the words in this paragraph?

File: working-groups/wg-tidb-sequence.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Members
+
+* [ailinkid](https://github.com/AilinKid)
+* [andrewdi](https://github.com/AndrewDi)
+
+## Contact
+
+* Slack: channel **#wg-tidb-sequence** in the
+  [tidbcommunity](https://pingcap.com/tidbslack) slack workspace.
```
Comment (gingerkidney) [inline working-groups/wg-tidb-sequence.md]:
```suggestion
  [tidbcommunity](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE) slack workspace.
```

Comment (AilinKid) [reply working-groups/wg-tidb-sequence.md]: Addressed.

File: working-groups/wg-tidb-sequence.md (hunk 2 above)
Comment (AilinKid) [reply working-groups/wg-tidb-sequence.md]: Addressed.

File: working-groups/wg-tidb-sequence.md (hunk 1 above)
Comment (AilinKid) [reply working-groups/wg-tidb-sequence.md]: Addressed.

**Comment by AilinKid** [comment]
@zz-jason @gingerkidney PTAL

File: working-groups/wg-tidb-sequence.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# TiDB Sequence Working Group
+
+## Goals
+
+Support Sequence in TiDB
+
+* Many customers could probably make do with using Auto_Increment as unique id generator,
+but in some cases, existing applications try to use sequences to fetch generated id back
```
Comment (dcalvin) [inline working-groups/wg-tidb-sequence.md]:
```suggestion
However, in some cases, existing applications try to use sequences to fetch the generated ID
```

File: working-groups/wg-tidb-sequence.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# TiDB Sequence Working Group
+
+## Goals
+
+Support Sequence in TiDB
```
[the remark this answers is not in the document]
Comment (dcalvin) [reply working-groups/wg-tidb-sequence.md]: Suggest putting this sentence behind the next two.

**Pull request merged**